.github/workflows/integration_tests.yml

```diff
@@ -45,6 +45,19 @@ jobs:
         with:
           path: dev/rucio
           fetch-depth: 0
+      - name: Set up Docker Buildx
```

Comment:
The steps for caching seem to be the same in both `autotest` and `integration_tests`. Can we implement this as a reusable workflow to reduce duplication?

For reference:
- https://docs.github.com/en/actions/sharing-automations/reusing-workflows
- https://docs.github.com/en/actions/sharing-automations/avoiding-duplication